rita/src/rita_client/dashboard/network_endpoints.rs

```diff
@@ -256,3 +256,14 @@ fn validate_config_value(s: &str) -> Result<(), ValidationError> {
         Ok(())
     }
 }
+
+pub fn get_wifi_config(
```

Comment:
[OCD Tier] You're calling the endpoint `/wifi_settings`, why is it `[...]_wifi_config` all of the sudden? Consistency helps keep things organized, having redundant names for the same (or closely coupled) thing is avoidably adding to the complexity of our code.

Reply:
this is how it was named before I ever touched it. Not sure it's worth trying to fix right now.